### PR TITLE
[web] Removed QR when metro isn't fired up

### PR DESCRIPTION
**Pull request by EvanBacon**
Trying to create a very simple path to starting web so things like code sandbox can use it easily.

### CODE REVIEW

File: packages/expo-cli/src/commands/start.js [hunk 1]
```diff
@@ -72,11 +78,13 @@ async function action(projectDir, options) {
   if (!nonInteractive && !exp.isDetachexped) {
     await TerminalUI.startAsync(projectDir, startOpts);
   } else {
```
Comment (ide) [inline packages/expo-cli/src/commands/start.js]:
```suggestion
  } else if (!options.webOnly) {
```

File: packages/expo-cli/src/commands/start.js [hunk 2]
```diff
@@ -72,11 +78,13 @@ async function action(projectDir, options) {
...
-    if (!exp.isDetached) {
-      log.newLine();
-      urlOpts.printQRCode(url);
+    if (!options.webOnly) {
```
Comment (ide) [inline packages/expo-cli/src/commands/start.js]: When web is enabled, do we print the URL of the site?

Comment (EvanBacon) [reply packages/expo-cli/src/commands/start.js]: we open the devtools with the link to open it. May be nice to have both though.

Comment (ide) [reply packages/expo-cli/src/commands/start.js]: I think it'd be useful to show the URL in the CLI too for people who don't have the web UI of Expo CLI enabled (they just use CLI) and also so we clarify which URLs are for which platforms, now that we have multiple URLs.

File: packages/expo-cli/src/commands/start.js [hunk 3]
```diff
@@ -71,7 +78,7 @@ async function action(projectDir, options) {
 
   if (!nonInteractive && !exp.isDetachexped) {
     await TerminalUI.startAsync(projectDir, startOpts);
-  } else {
+  } else if (!options.webOnly) {
```
Comment (ide) [inline packages/expo-cli/src/commands/start.js]: Maybe change:
```
log(`Your app is running at ${chalk.underline(url)}`)
```
to 
```
log(`Your native app is running at ${chalk.underline(url)}`);
```

**Pull request merged**